src/apollo/types/types.ts

```diff
@@ -63,6 +63,7 @@ export interface IProject {
 		name?: EProjectStatus;
 	};
 	updatedAt: string;
+	latestUpdateCreationDate: string;
```

Comment:
Is it not better to make this field optional? (I think this help safer deploy and prevent crashes)

Reply:
I have added `latestUpdateCreationDate` to project table, just like updatedAt above that is required field.

Reply:
I made that optional